cchecker.py

```diff
@@ -92,6 +92,10 @@ def main():
     if args.download_standard_names:
         download_cf_standard_name_table(args.download_standard_names)
 
+    if len(args.dataset_location) == 0:
+        parser.print_help()
+        return 0
```

Comment:
:+1: I just wonder if the return code should really be `0`. In theory no args is an error and printing the `help` without the user requesting it is a "hint" and not an expected result.